agr_indexer/src/main/java/org/alliancegenome/indexer/DiseaseAnnotationRESTInterface.java

```diff
@@ -0,0 +1,31 @@
+package org.alliancegenome.indexer;
+
+import org.alliancegenome.curation_api.model.entities.AGMDiseaseAnnotation;
+import org.alliancegenome.curation_api.model.entities.AlleleDiseaseAnnotation;
+import org.alliancegenome.curation_api.model.entities.GeneDiseaseAnnotation;
+import org.alliancegenome.curation_api.response.SearchResponse;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+
+@Path("/api")
+public interface DiseaseAnnotationRESTInterface {
```

Comment:
@cmpich we should be able to use the interfaces from the library. And you can configure the auth headed in the configClient vs on every call.